### PR TITLE
Add IO() properties and IsItemActive()

### PR DESCRIPTION
**Pull request by supudo**
Small PR for some IO() props and IsItemActive().

### CODE REVIEW

File: IO.go [hunk 1]
```diff
@@ -30,6 +30,48 @@ func (io IO) WantTextInput() bool {
...
+}
+
+// MetricsActiveAllocations returns number of active allocations, updated by MemAlloc/MemFree
+// based on current context. May be off if you have multiple imgui contexts. 
+func (io IO) MetricsActiveAllocations() int {
+	return int(C.iggMetricsActiveAllocations(io.handle))
+}
+    
+// MouseDelta returns the mouse delta movement . Note that this is zero if either current or previous position
+// are invalid (-FLT_MAX,-FLT_MAX), so a disappearing/reappearing mouse won't have a huge delta.
```
Comment (dertseha) [inline IO.go]: How does `-FLT_MAX` translate to Go? What would be the corresponding constants here?

**Comment by dertseha** [comment]
Looks almost good, the one comment might have to be translated to Go.

File: IO.go (hunk 1 above)
Comment (supudo) [reply IO.go]: Replaced it with the Go equivalent, which is math.MaxFloat32.

**Comment by dertseha** [comment]
Great, thank you for your contribution!

**Pull request merged**